app/src/main/java/de/danoeh/antennapod/fragment/AllEpisodesFragment.java

```diff
@@ -498,4 +525,43 @@ void removeNewFlagWithUndo(FeedItem item) {
         snackbar.show();
         h.postDelayed(r, (int) Math.ceil(snackbar.getDuration() * 1.05f));
     }
+
+    private void showFilterDialog() {
```

Comment:
Extracted a FilterDialog class.

Reply:
Could you please push your changes, so I can have a look? :)

Reply:
Committed. Sorry, I have very limited time these days

Reply:
I did not want to stress you. Because you commented that the changes are done, I assumed that it was already ready and you simply forgot to push. Did not want to run into the situation that we both wait for each other :D

Reply:
Found the time finally now :-) No stress involved... want to see the two features in an F-Droid version as soon as possible myself :-D